ios/Exponent/Supporting/Info.plist

```diff
@@ -31,6 +31,7 @@
 			<array>
 				<string>exp</string>
 				<string>exps</string>
+				<string>fb1696089354000816</string>
```

Comment:
If so, could we remove https://github.com/expo/expo/blob/0ff9a3f7d08a177fe51e822425ddff1af485372c/packages/expo-facebook/ios/EXFacebook/EXFacebookAppDelegate.m#L29-L32?

Reply:
Maybe the method swizzling is the cause of all these problems in the first place? 🤔

Reply:
^ removed the swizzling code in the most recent commits